pb_arithmetic.js

```diff
@@ -0,0 +1,55 @@
+let snackServings = 50;
+console.log(snackServings);
+let guests = 15;
+console.log(guests);
+
+console.log(guests - snackServings);
+console.log(snackServings - guests);
+console.log(snackServings * guests);
+console.log(snackServings / guests);
+
+let drinks = 10;
+console.log(drinks);
+
+console.log((snackServings * guests) / drinks);
+let resultOne = 75;
+console.log(resultOne);
+
+let monkeys = 10;
+console.log(monkeys);
+let bananas = 9;
+console.log(bananas);
+console.log(monkeys / bananas);
+
+let monkeyfights = 20;
+console.log((monkeys + bananas) * monkeyfights);
+
+let resultTwo = 380;
+
+// Increment (++)
+// Decrement (--)
+console.log(++monkeys);
+
+console.log(--bananas);
+
+/*
+let x = 3;
+const y = x++;
+
+console.log(`x:${x}, y:${y}`);
+    expected output: "x:4, y:3"
+
+let a = 3;
+const b = ++a;
+
+console.log(`a:${a}, b:${b}`);
+    expected output: "a:4, b:4"
+*/
+
+console.log(--bananas - ++monkeys);
+
+let happiness = -5;
+console.log(happiness + monkeyfights);
+
+console.log(resultOne / resultTwo);
```

Comment:
//11. Print the remainder (%) when resultOne is divided by resultTwo to the console.

Reply:
console.log(resultOne % resultTwo);